### PR TITLE
add icon and view

### PR DESCRIPTION
**Pull request by ghost**
## Related Tickets
- [#Task 74506: Navigation](https://edu-redmine.sun-asterisk.vn/issues/74506)
## WHAT
- add icon and view 
## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/blob/master/eng/android/coding_convention_android_kotlin.md | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>
## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(Other notes)*

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -5,13 +5,22 @@
...
 
-    <TextView
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="Hello World!"
-        app:layout_constraintBottom_toBottomOf="parent"
+    <androidx.viewpager2.widget.ViewPager2
+        android:id="@+id/pager2"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
```
Comment (ghost) [inline app/src/main/res/layout/activity_main.xml]: chỗ này bỏ vô dimen nha ông

File: app/src/main/res/layout/fragment_home.xml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        android:paddingTop="@dimen/dimen_10"
+        android:paddingBottom="@dimen/dimen_12"
+        android:src="@drawable/ic_instagram_logo"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/recycleView"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
```
Comment (ghost) [inline app/src/main/res/layout/fragment_home.xml:20]: .

File: app/src/main/res/layout/item_collection.xml [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+            android:layout_height="wrap_content"
+            android:textSize="@dimen/dimen_13sp"
+            android:textStyle="bold"
+            tools:text="joshua_l" />
+
+        <TextView
+            android:id="@+id/tvLocation"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="1dp"
```
Comment (ghost) [inline app/src/main/res/layout/item_collection.xml]: này nữa nè

File: app/src/main/res/layout/item_collection.xml [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+        android:layout_marginTop="@dimen/dimen_5"
+        android:textSize="@dimen/dimen_13sp"
+        android:textStyle="bold"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/imgLike"
+        tools:text="joshua_l" />
+
+    <TextView
+        android:id="@+id/tvContent"
+        android:layout_width="0dp"
```
Comment (ghost) [inline app/src/main/res/layout/item_collection.xml]: .

**Review by ghost — APPROVED** [review]

**Review by daolq-2712 — APPROVED** [review]
OK!

**Pull request merged**